README.md

```diff
@@ -5,6 +5,18 @@ designed specifically for Scala. Project is currently in pre-release stage.
 [Follow us on twitter to be first to know about upcoming
 updates.](http://twitter.com/scala_native)
 
+## How to build?
+
+This requires SBT to be installed on your machine. Refer to [this link](http://www.scala-sbt.org/release/docs/Setup.html) for instructions.
+
+```
+$ git clone https://github.com/scala-native/scala-native.git --recursive
+$ sbt rtlib/publishLocal nscplugin/publishLocal
+$ sbt package
+```
+
+If you face any issues with your development environment setup, refer to [developer guide](DEVELOP.md)
```

Comment:
All the docs should go in `docs/` subfolder. E.g. this one can be `/docs/develop.md`